aiida/utils/fixtures.py

```diff
@@ -362,29 +375,21 @@ def destroy_all(self):
         if 'profile' in self._backup:
             backend_settings.AIIDADB_PROFILE = self._backup['profile']
 
-    @staticmethod
-    def __clean_db_django():
-        from aiida.backends.djsite.db.testbase import DjangoTests
-        DjangoTests().clean_db()
-
-    def __clean_db_sqla(self):
```

Comment:
Is this stuff no longer necessary, this creation of a new_user object?

Reply:
It's now done at the ORM level in the `aiida/backends/testimplbase.py` `insert_data` so it's backend independent...this is the way, ideally, all these things would be but we can't delete everything from ORM function calls.